stock_average_daily_sale/models/stock_average_daily_sale.py

```diff
@@ -250,6 +315,28 @@ def _create_materialized_view(self):
                     FROM deliveries_last
                     GROUP BY product_id, warehouse_id, standard_deviation, nbr_days, date_from, date_to, config_id
                 ),
+                averages_return AS(
+                    SELECT
+                        row_number() over (order by product_id) as id,
+                        concat(warehouse_id, product_id)::integer as window_id,
```

Comment:
This looks dangerous to me, but perhaps context assures that there's no 11th warehouse?
```
select concat(11, 2)::integer = concat(1, 12)::integer;
-> true
```

Reply:
Good catch! I suppose it should be fixed on the OCA as well and here only for the returns part, right @sebalix? What would be the recommended solution for that? Is something like `concat(warehouse_id, 0, product_id)` enough?

Reply:
Since record IDs are always integers, easy way to avoid conflicts is to use a non-integer character as the divider and have the combination ID be a string (unless there is a reason that this combination ID needs to be an integer?)
For example:
`concat(warehouse_id, '-', product_id)`

Reply:
I pushed a fix here for the returns part and on OCA upstream https://github.com/OCA/stock-logistics-reporting/pull/357